giles-eco/src/main/resources/velocitytemplates/email/accountApproved.vm

```diff
@@ -0,0 +1,5 @@
+Hi $name:
+
+Your Giles account has been approved by the admin! You can now login to Giles at http://$giles_url/ with $userName.
```

Comment:
Let's remove the "by the admin" part. It doesn't matter who approved it, just that it was approved.